apps/glusterfs/app_block_volume_test.go

```diff
@@ -0,0 +1,565 @@
+//
```

Comment:
There should be tests added to check the cluster settings. If the cluster has Block:false, then this should return XXX according to the API